js_modules/dagster-ui/packages/ui-core/src/asset-graph/useAssetGraphExplorerFilters.oss.tsx

```diff
@@ -3,20 +3,21 @@ import {useAssetCatalogFiltering} from 'shared/assets/useAssetCatalogFiltering.o
 
 import {AssetGraphViewType, GraphNode} from './Utils';
 
-type Props = {
+export type Props = {
   nodes: GraphNode[];
-  clearExplorerPath: () => void;
-  explorerPath: string;
+  clearAssetSelection: () => void;
+  setAssetSelection: (selection: string) => void;
```

Comment:
This seems to not use `setAssetSelection`?

Reply:
yeah its for cloud 😬